templates/citizen_project/show_base.html.twig

```diff
@@ -41,15 +41,21 @@
 
 {% block citizen_project_content %}
     {% if is_administrator %}
-        <div class="b__nudge--top">
-            <a href="{{ path('app_citizen_project_manager_edit', {slug: citizen_project.slug}) }}"
-               class="text--body link--newblue">
-                Modifier ce projet
-            </a>
-            <a href="{{ path('app_citizen_project_image_generator') }}" class="text--body link--newblue">
-                Créer des images pour nos réseaux sociaux
-            </a>
-        </div>
+        <nav class="b__nudge--top">
+            <ul class="list__links list__links--margin-right">
+                <li>
+                    <a href="{{ path('app_citizen_project_manager_edit', {slug: citizen_project.slug}) }}"
```

Comment:
There should be white spaces inside brackets: 
`{ slug: citizen_project.slug }`
(same in other templates of this PR, not a big deal)